src/components/Dashboard/Health/strings.js

```diff
@@ -0,0 +1,4 @@
+export const HEALTHY= 'OCS is Healthy';
```

Comment:
these values are specific for Storage dashboard

Reply:
What do you suggest the messages should be? Is 'System is healthy' fine?

Reply:
just keep these strings in Storage Overview, we do not need them in generic Dashboard Health. When you will want to use Dashboard Health you will specify the message